test/llvm/fma/FmaHardwareSpecialization.skipif

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+
+import os
+import subprocess
+
+CHPL_HOME = os.getenv('CHPL_HOME')
+CHPL_LLVM_CONFIG=None
+CHPL_TARGET_ARCH=None
+CHPL_TARGET_CPU=None
+CHPL_TARGET_PLATFORM=None
+CHPL_TARGET_COMPILER=None
+chplenv = subprocess.check_output([CHPL_HOME + "/util/printchplenv",
+                                   "--all", "--internal", "--simple"])
+
+# Get some 'printchplenv' values.
+for line in chplenv.splitlines():
+  line_str = str(line, encoding='utf-8', errors='surrogateescape')
+  if not line_str.count('=') == 1: continue
+  (k, v) = line_str.split('=', 2)
+  if k == 'CHPL_LLVM_CONFIG': CHPL_LLVM_CONFIG=v
+  if k == 'CHPL_TARGET_ARCH': CHPL_TARGET_ARCH=v
+  if k == 'CHPL_TARGET_CPU': CHPL_TARGET_CPU=v
+  if k == 'CHPL_TARGET_PLATFORM': CHPL_TARGET_PLATFORM=v
+  if k == 'CHPL_TARGET_COMPILER': CHPL_TARGET_COMPILER=v
+
+# Confirm we're compiling with LLVM and are on x64.
+Ok = CHPL_TARGET_COMPILER == 'llvm'
+Ok = Ok and CHPL_TARGET_ARCH == 'x86_64'
+Ok = Ok and (CHPL_TARGET_CPU != 'none' and CHPL_TARGET_CPU != 'unknown')
+Ok = Ok and (CHPL_TARGET_PLATFORM == 'linux64' or
+             CHPL_TARGET_PLATFORM == 'darwin')
+
+# Make sure that our x64 CPU reports the 'fma' extension (or else don't
+# bother testing, as LLVM probably won't generate a hardware instruction).
```

Comment:
Maybe adjust the above section to also check that `CHPL_LAUNCHER==none` since otherwise the system we are running on now might differ from where the program will run